packages/p2p/src/stores/my-ads-store.js

```diff
@@ -33,6 +35,8 @@ export default class MyAdsStore extends BaseStore {
     @observable is_quick_add_modal_open = false;
     @observable is_table_loading = false;
     @observable is_loading = false;
+    @observable should_switch_ad_rate = false;
+    @observable is_switch_modal_open = false;
```

Comment:
Can you please alphabetize these two new observables like the rest of them.

Reply:
Re-arranged them